plugins/graphql-kotlin-gradle-plugin/src/integration/download-sdl-timeout/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+import com.expediagroup.graphql.plugin.gradle.config.TimeoutConfiguration
+import com.expediagroup.graphql.plugin.gradle.tasks.GraphQLDownloadSDLTask
+
+plugins {
+  kotlin("jvm") version "1.7.21"
```

Comment:
we could potentially load this from version catalog (but would have to update the test logic to copy the toml file - will look into this later)